worker-sandbox/src/counter.rs

```diff
@@ -19,18 +20,35 @@ impl DurableObject for Counter {
         }
     }
 
-    async fn fetch(&mut self, _req: Request) -> Result<Response> {
+    async fn fetch(&mut self, req: Request) -> Result<Response> {
         if !self.initialized {
             self.initialized = true;
             self.count = self.state.storage().get("count").await.unwrap_or(0);
         }
 
         self.count += 10;
         self.state.storage().put("count", self.count).await?;
+
+        if req.path().contains("alarm") {
+            // Set an alarm to trigger in 500 ms:
+            let now = Utc::now();
+            self.state
+                .storage()
+                .set_alarm(now + chrono::Duration::milliseconds(500))
+                .await?;
+        }
+
         Response::ok(&format!(
             "[durable_object]: self.count: {}, secret value: {}",
             self.count,
             self.env.secret("SOME_SECRET")?.to_string()
         ))
     }
+
+    async fn alarm(&mut self) -> Result<Response> {
```

Comment:
We should move the alarm logic to a different durable object and then run tests on that, but something like this is what I would do. We'd also need to update the test in `worker-sandbox/tests/requests.rs`.